### PR TITLE
RHD-1160 Docker README

### PR DESCRIPTION
**Pull request by ollyjshaw**

### CODE REVIEW

**Comment by ollyjshaw** [comment]
retest this please

File: _docker/README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+bundle exec ./control.rb --run-the-stack
+```
+
+This will allow you to work with any of the pages generated by awestruct. If you want to quit, then `CTRL-C` to stop.
+
+## Running the acceptance tests
+Every commit will be acceptance tested once a Pull Request is open. To run this via docker then the command is :
+
+```bash
+bundle exec ./control.rb --acceptance_test_docker
```
Comment (paulrobinson) [inline _docker/README.md]: When I run this I see the following when accessing: http://docker:32768/

      Internal Server Error

      undefined method `[]' for nil:NilClass
      WEBrick/1.3.1 (Ruby/2.1.2/2014-05-08) at docker:32768

Want to take a look when you get in the office tomorrow?

**Comment by ollyjshaw** [comment]
@paulrobinson I get this occasionally too. Often when containers have not shut down properly. We should take another look and file a Jira ticket if needed.

**Comment by paulrobinson** [comment]
Eventually I got this working. Can you add the workarounds to the docs, that you did on my machine?

**Comment by ollyjshaw** [comment]
retest this please

**Comment by ollyjshaw** [comment]
retest this please

**Pull request closed without merging**